MAVProxy/modules/mavproxy_chat/__init__.py

```diff
@@ -55,6 +62,53 @@ def cmd_chat(self, args):
     def show(self):
         self.chat_window.show()
 
+    # handle mavlink packet
+    def mavlink_packet(self, m):
+        if m.get_type() == 'COMMAND_ACK':
```

Comment:
Often worth creating an `m_type` in these straight-up.  `get_type` can be surprisingly expensive when called multiple times in `mavlink_packet`, and `elif` statements tend to accrete in `mavlink_packet` calls.

Reply:
OK noted.  I did notice that in other modules but in this case it only checks it once so I avoided the local variable for now anyway.